server/channels/api4/file_test.go

```diff
@@ -805,10 +805,91 @@ func TestGetFile(t *testing.T) {
 	_, resp, err = client.GetFile(context.Background(), fileId)
 	require.Error(t, err)
 	CheckUnauthorizedStatus(t, resp)
+}
+
+func TestGetFileAsSystemAdmin(t *testing.T) {
```

Comment:
I would be more at ease if we also refactor `TestGetFile` to follow this same approach.